Frontend/src/components/forms/ContactForm.jsx

```diff
@@ -147,11 +147,7 @@ export default function ContactForm({
         open={openModal}
         closeAfterTransition
         // submit on Enter key
-        onKeyDown={(e) => {
-          if (e.key === "Enter") {
-            submit();
```

Comment:
Kako to da više nesubmitamo na enter?

Reply:
Znaci kada ti ispunjavas formu i nisi zadovoljio validacijska pravila onda je _add/update_ button disablean (grayed out) i logicno ne mozes ga kliknuti. Ako ovo postoji onda cim otvoris formu i kliknes enter se zove submit bez obzira dal si ti zadovoljio validacijska pravila.

Budem promijenio umjesto da taj kod izbrisem da dodam jos jedan if koji provjerava dal su zadovoljena sva validacijska pravila pa tek ako jesu da se zove submit.